packages/grafana-ui/src/types/time.ts

```diff
@@ -11,11 +11,29 @@ export interface TimeRange {
   raw: RawTimeRange;
 }
 
+export interface AbsoluteTimeRange {
+  from: number;
+  to: number;
+}
+
 export interface IntervalValues {
   interval: string; // 10s,5m
   intervalMs: number;
 }
 
+export interface TimeZone {
+  raw: string;
+  isUtc: boolean;
+}
+
+export const parseTimeZone = (raw: string): TimeZone => {
```

Comment:
Maybe introduce a type for that?
```suggestion
type TimeZone = 'utc' | 'browser' | 'blabla';
export const parseTimeZone = (raw: TimeZone): TimeZone => {
```

Reply:
Decided to solve this later since it requires a lot of changes. Updated #16806